test/users/ferguson/badcallerror.good

```diff
@@ -1 +1 @@
-badcallerror.chpl:3: error: type mismatch in assignment from function ():int(64) to int(64)
+badcallerror.chpl:3: error: Cannot assign to int(64) from chpl__fcf_type_void_int64_t
```

Comment:
Since this is a future, the output isn't expected to match. I think it should be 
```
badcallerror.chpl:3: error: Cannot assign to int(64) from function()
```

Reply:
@mppf Sir, I think we need to do the update in future file. This is .good file

Reply:
In our testing system, a .future records something that isn't working now. This future is about how the compiler prints out the type name for a first-class function. The .good file should not match the current output. (If it did, we could make it a regular test instead of a .future). Instead, it should match what would be better output. It should "Fail" (but the testing system should know it's a future).

Reply:
Thanks sir. Also, I have updated the rest of the files as you asked.